nullaway/src/main/java/com/uber/nullaway/handlers/BaseNoOpHandler.java

```diff
@@ -53,7 +53,7 @@
  * interface. Additionally, we can add extensibility points without breaking existing handlers, as
  * long as we define the corresponding No-Op behavior here.
  */
-abstract class BaseNoOpHandler implements Handler {
+public abstract class BaseNoOpHandler implements Handler {
```

Comment:
Why are we making `BaseNoOpHandler` accessible outside of `handlers`. Is the issue that package private doesn't extend to sub-packages?

Reply:
Yes.